src/style.css

```diff
@@ -0,0 +1,123 @@
+.todo-list {
+  display: flex;
+  justify-content: center;
+}
+
+.task-outer-container {
+  border: 3px solid #ecebea;
+  flex-basis: 70vw;
+}
+
+.title-container {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid #ecebea;
+}
```

Comment:
I noticed some colors are used several times, you might want to use global variables, it improves the maintainability of your code. Here's an example of how to use global variables:

Include a root class at the beginning of your CSS.
`:root {
  --red: #b00;
  --blue: #00b;
  --fullwidth: 100%;
}`

Then, wherever you wish to use that color, include it like this: 

`h1 {
  color: var(--red);
}
#MyText {
  color: var(--blue);
  width: var(--fullwidth);
}`

Reply:
Changed successfully, thanks!